spring-boot/admin/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,44 @@
+spring:
+  data:
+    rest:
+      base-path: /api
+logging:
+  file: /tmp/coatrack.log
+  level:
+    nl: DEBUG
+    org: INFO
+    com: DEBUG
+ygg:
+  admin:
+    server:
+      url: http://localhost:8080
+    servicecovers:
+      path: /tmp
+      url: http://localhost/coatrack-servicecovers/
+
+    payment.commision:
+      variable: 50
+      fix: 1
+      bookKeeping.contact: email@example.com
+    api-base-url-for-gateway: https://coatrack.eu/api/
+    call-request-fee: 0
+    database:
+      insertSampleDataOnStartup: true
+    gettingStarted:
+      consumer:
+        testService:
+          provider.username: BrunoSilvaATB
+          uriIdentifier: getting-started-service-consumer
+  mail:
+    verification.server.url: http://localhost:8080
```

Comment:
To allow the build to run without specific local mvn settings, we should also add the following here:

` sender.user: example-mail@example.com`
` sender.password: examplePassword`

Reply:
Done and working well.